build.gradle.kts

```diff
@@ -176,7 +174,11 @@ tasks.register<JacocoReport>("jacocoTestReport") {
     dependsOn(subprojects.map { it.tasks.getByName("jacocoTestReport") })
     val main = subprojects.map { it.sourceSets.named("main").get() }
     additionalSourceDirs(files(main.map { it.allSource.srcDirs }))
-    additionalClassDirs(files(main.map { it.output }))
+    additionalClassDirs(
+        files(
+            main.map { it.output }.map { it.flatMap { file -> fileTree(file) { exclude(exclusions) } } }
+        )
+    )
```

Comment:
Do we need to do this?  This task just aggregates the execution data from the subprojects, so haven't the exclusions already have been applied there?